tests/test_quantization/lifecycle/test_apply.py

```diff
@@ -97,7 +97,7 @@ def test_serialize_config_tinyllama():
     assert serialized_config.config_groups["group_1"].input_activations is not None
     assert serialized_config.quantization_status == QuantizationStatus.FROZEN
     assert serialized_config.format == "dense"
-    assert serialized_config.quant_method == "sparseml"
+    assert serialized_config.quant_method == "compressed-tensors"
```

Comment:
Can we have this, in general, set as a flag?

```suggestion
    from ... import PACKAGE_NAME
    ...
    assert serialized_config.quant_method == _PACKAGE_NAME
```

Reply:
Done!